app/models/taxonomy.rb

```diff
@@ -48,6 +48,7 @@ class Taxonomy < ActiveRecord::Base
   after_create :assign_default_templates
 
   scoped_search :on => :description, :complete_enabled => :false, :only_explicit => true
+  scoped_search :on => :id
```

Comment:
imho this should be only_explicit with no auto completer, as ID is usually not user friendly.